File: python/fetch_tables.py
import boto3
# Script to ensure that data is in table and able to be accessed. 

def _get_table_data():
    dynamodb = boto3.resource('dynamodb')
    table_name = 'rentable_properties'
    table = dynamodb.Table(table_name)
    data = table.scan()
    return data['Items']
print(_get_table_data())

File: ruby/fetch_tables.rb
require 'aws-sdk-dynamodb'

def _get_table_data()
    dynamodb = Aws::DynamoDB::Resource.new(region: 'us-east-1')
    table_name = 'properties'
    table = dynamodb.table(table_name)
    data = table.scan()
    data[:items]
end

puts _get_table_data